code/game/machinery/suit_storage_unit.dm

```diff
@@ -1211,11 +1222,22 @@
 		helmet.item_state_slots = H.item_state_slots
 		qdel(H)
 
-	if(suit)
+	if(suit && target_department != "No Change")
 		var/obj/item/clothing/S = new parent_suit
 		suit.name = "refitted [initial(parent_suit.name)]"
 		suit.desc = initial(parent_suit.desc)
 		suit.icon_state = initial(parent_suit.icon_state)
 		suit.item_state = initial(parent_suit.item_state)
 		suit.item_state_slots = S.item_state_slots
 		qdel(S)
+
+		//can't believe I forgot to fix this- now helmets will properly cycle if they're attached to a suit -KK
+		if(suit.helmet && target_department != "No Change")

```

Comment:
This needs one tab removed to be on-line with the other if checks, I think.

Reply:
No, because if no suit is fitted into the cycler it'll try to read the mounted helmet value from a `null` suit, which will (harmlessly) runtime.